Packet++/header/SomeIpLayer.h

```diff
@@ -133,7 +133,7 @@ class SomeIpLayer : public Layer
 	/**
 	 * Get a pointer to the basic SOME/IP header. Notice this points directly to the data, so every change will change
 	 * the actual packet data
-	 * @return A pointer to the @ref someiphdr

```

Comment:
because of the failing test from the Doxygen Documentation pipeline (also line 90)...not sure why...it already worked with the SOME/IP-TP branch...for me the `@ref someiphdr` looks ok:
`Generating docs for compound pcpp::radius_hea/tmp/cirrus-ci-build/Dist/header/SomeIpLayer.h:136: error: unable to resolve reference to `someiphdr' for \ref command (warning treated as error, aborting now)`

Reply:
Hmm I'm not sure why you get this error 🤔 
I'll take a look...